rfcs/text/0000-deprecate-log-original.md

```diff
@@ -0,0 +1,116 @@
+# 0000: Name of RFC
+<!-- Leave this ID at 0000. The ECS team will assign a unique, contiguous RFC number upon merging the initial stage of this RFC. -->
+
+- Stage: **0 (strawperson)** <!-- Update to reflect target stage. See https://elastic.github.io/ecs/stages.html -->
+- Date: **TBD** <!-- The ECS team sets this date at merge time. This is the date of the latest stage advancement. -->
+
+<!--
+As you work on your RFC, use the "Stage N" comments to guide you in what you should focus on, for the stage you're targeting.
+Feel free to remove these comments as you go along.
+-->
+
+<!--
+Stage 0: Provide a high level summary of the premise of these changes. Briefly describe the nature, purpose, and impact of the changes. ~2-5 sentences.
+-->
+
+This RFC supersedes issue #841 which implies breaking changes therefore the RFC Process is indicated
+
+The request is to consolidate `log.original` and `event.original` by removing `log.original`, since these are almost equivalent in nature. (One) justification for preserving `event.original` is that not all events are logs.
+
+Breaking changes may be:
+
+- requirement to alias `log.original` for current users of this field
+- TBD if there exist current users of fields with distinct content/meaning in a common index mapping
```

Comment:
Our integrations team will need to audit our supported integrations, but I just noticed our Google Cloud integration includes a var.keep_original_message (disabled by default) which populates `log.original`. Will need to adjust the module to populate `event.original` instead.